src/trainer_card.c

```diff
@@ -59,61 +77,16 @@ struct TrainerCardData
     /*0x000E*/ u8 var_E;
     /*0x000F*/ u8 var_F;
     /*0x0010*/ bool8 hasTrades;
-    /*0x0011*/ u8 hasBadge[BADGE_COUNT];
-    /*0x0019*/ u8 var_19[4][0xD];
-    /*0x004D*/ u8 var_4D[0x46];
-    /*0x0093*/ u8 var_93[0x46];
-    /*0x00D9*/ u8 var_D9[0x46];
-    /*0x011F*/ u8 var_11F[0x46];
-    /*0x0165*/ u8 var_165[0x46];
-    /*0x01AB*/ u8 var_1AB[0x46];
-    /*0x01F1*/ u8 var_1F1[0x46];
-    /*0x0237*/ u8 var_237[0x46];
-    /*0x027D*/ u8 var_27D[0x46];
-    /*0x02C3*/ u8 var_2C3[0x46];
-    /*0x0309*/ u8 var_309[0x46];
-    /*0x034F*/ u8 var_34F[0x46];
+    /*0x0011*/ bool8 hasBadge[BADGE_COUNT];
+    /*0x0019*/ u8 var_19[4][13];
+    /*0x004D*/ u8 strings[12][70];
```

Comment:
add a `TRAINER_CARD_STRING_COUNT` entry to the enum above, and replace 12 with it